content/resources/user/gen3-client.md

```diff
@@ -137,55 +184,56 @@ Finished downloads/24d0be10-d164-48ad-aafa-9fcaac682df9 2570240 / 2570240 bytes
 ```
 
 * * *
-## 6) Provide a Manifest File for Bulk Upload
+## 7) Provide a Manifest File for Bulk Upload
 * * *
-Users, who need to semi-automate the upload of files, can modify the following shell script to fit their needs. This shell script can be run from the command-line:
+Users can automate the upload of a bulk of files by providing an upload manifest. Ideally the upload manifest is the same as the download manifest that can be generated automatically as described in the previous section. However, user can also create a "minimal" upload manifest on their own if needed. A valid "minimal" upload manifest file is a JSON file that could only contains many `object_id` fields. The value of each `object_id` field is the GUID of a data file that is going to be submitted. In this mode, for now we are assuming all the data files to be uploaded have the same filenames as their GUIDs.
```

Comment:
"A valid 'minimal' upload file is a JSON file that only contains `object_id` fields."